lago/vm.py

```diff
@@ -99,6 +99,10 @@ def stop(self):
             with LogTask('Destroying VM %s' % self.vm.name()):
                 self.libvirt_con.lookupByName(self._libvirt_name(), ).destroy()
 
+    def restart(self):
```

Comment:
I think it would be much easier to implement all of the above by adding a '--reboot' parameter to do_stop() , which will also start the VM. Most of the logic is already implemented in start/stop.